### DISCUSSION TITLE
Nested Virtualization on Ryzen Cpu

### DISCUSSION THREAD

**Discussion by saksham77ang** [notAperson535/OneClick-macOS-Simple-KVM#125] [Show and tell]
Hey @notAperson535 Thank you for making running mac via windows so easy. But I read that you have stated ryzen cpu's cant use nested virtualization and found [This](https://community.amd.com/t5/gaming-discussions/amd-help-microsoft-with-nested-virtualization/td-p/170754) article. I'm noob so hardly understand any of it. Hope you can solve the ryzen issue with this and if so please reply so I can go ahead with my installation as I'm stuck with a ryzen 5 3500u...

**Comment by notAperson535** [comment]
@saksham77ang you can just try it with your Ryzen. I haven’t tested it before and people have had issues in the past but it might be fixed now. Let me know how it goes.

**Reply by saksham77ang** [reply]
Can you share the vm name I can't see it. I downloaded ventura if it matters.... As the code on the page needs a vm name

**Reply by notAperson535** [reply]
It should just work without it, ignore the guide and what I said on the website for now and just let me know how it goes

**Reply by saksham77ang** [reply]
![image](https://github.com/user-attachments/assets/670ce616-5d3c-4b70-8e85-f96945a40a55)

**Reply by notAperson535** [reply]
@saksham77ang Just ignore that guide and follow the normal one I have on the website. If it doesn’t work we can look into the guide you sent.
